placement_home.html

```diff
@@ -123,7 +123,7 @@ <h3 style="font-family: 'Merriweather', serif;", cursive;>INDIAN INSTITUTE OF TE
     <section class="row text-center placeholders">
         <div class="col-6 col-sm-3 placeholder">
             <a href="#mess_dir"><img src="http://iitism.ac.in/assets/img/admin/dt.jpg" width="200" height="200" class="img-fluid rounded-circle" alt="Generic placeholder thumbnail"></a>
```

Comment:
Thanks for the PR, could you also update the pic of director? Possibly bring it offline?

Reply:
Welcome, @prakharvk  ! you are the first contributor to the project.